### PR TITLE
Make proposal of mdl.js

### PR DESCRIPTION
**Pull request by lukasjhan**
Hi guys :)

Add the project proposal to add mdl.js to the Open Wallet Foundation.

### CODE REVIEW

File: projects/mdl-js.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+There is only one mdl implementation from Auth0 labs, but they don't support React Native which is most crucial for mobile wallets. This project aims to provide a platform agnostic implementation of the MDL specification.
+
+I guess the main reason for not providing a React Native is that core dependencies like CBOR, COSE and X509 libraries are not available for React Native. So I decided to make them all platform agnostic.
+
+To provide mdl features to Biggest Wallet SDKs like credo or veramo, direction of the project will be:
+
+- Provide a platform agnostic implementation of the MDL specification
+- Bring your own crypto
+- Modular approach
+- Provide platform sepcific helper implementations for developers to use easily
```
Comment (tkuhrt) [inline projects/mdl-js.md:22]:
```suggestion
- Provide platform specific helper implementations for developers to use easily
```

File: projects/mdl-js.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+@aceshim Ace Shim
+
+# Project License
+
+Apache 2.0
+
+# Source Control
+
+https://github.com/lukasjhan/mdoc
```
Comment (tkuhrt) [inline projects/mdl-js.md:42]: I noticed that DCO does not exist on the current repository. See [Fixing DCO](https://github.com/openwallet-foundation/project-proposals?tab=readme-ov-file#fixing-dco-on-existing-repositories) for more information. We will need this corrected prior to bringing this code into the OpenWallet Foundation. We can also create a new repo and have you squash/merge if that is preferred.

Comment (lukasjhan) [reply projects/mdl-js.md:42]: I'll fix DCO :)

Comment (lukasjhan) [reply projects/mdl-js.md:42]: @tkuhrt I just finished add sign-off to all my commit :)

**Review by skounis — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]
Approved at the June 12, 2024 meeting

**Pull request merged**